specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/datadog.json

```diff
@@ -0,0 +1,2503 @@
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "SingleSignOn"
+        ],
+        "summary": "Gets the datadog single sign-on resource for the given Monitor.",
+        "operationId": "SingleSignOnConfigurations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "description": "Configuration name",
+            "in": "path",
+            "name": "configurationName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DatadogSingleSignOnResource"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SingleSignOnConfigurations_Get": {
+            "$ref": "./examples/SingleSignOnConfigurations_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Datadog/monitors/{monitorName}/monitoredSubscriptions": {
+      "get": {
+        "tags": [
+          "monitoredSubscriptions"
+        ],
+        "summary": "List the subscriptions currently being monitored by the Datadog monitor resource.",
+        "operationId": "MonitoredSubscriptions_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitorNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/MonitoredSubscriptionProperties"
```

Comment:
>MonitoredSubscriptionProperties [](http://example.com/codeflow?start=37&length=31)

for List API shouldn't it return a list response? please fix it.  also add

 "x-ms-pageable": {
          "nextLinkName": "nextLink"
        },

Reply:
This is already merged in private repo and the customers are using it. If we change it, it will be a breaking change. We can look into this as a part of new version when we incorporate other changes on how to maintain both the versions.

Reply:
@arnavshringi , Could you please share the private repo PR link for the change?

Reply:
Can you check this and add x-ms-pageable. Keep it as null if your API do not support it yet. 
[https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=x-ms-pageable#oapi009-always-add-x-ms-pageable-to-list-calls](https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html?q=x-ms-pageable#oapi009-always-add-x-ms-pageable-to-list-calls)

Reply:
Added @mukundnigam